### PR TITLE
Use scopes for funded and activated artists

### PR DESCRIPTION
**Pull request by Katee**
Cleans up solution for https://github.com/FireflyArtsCollective/ffagc/issues/91

### CODE REVIEW

File: app/models/artist.rb [hunk 1]
```diff
@@ -16,28 +16,20 @@ class Artist < ActiveRecord::Base
 
   validates_confirmation_of :password, on: :create
 
+  scope :activated, -> { where(activated: true) }
+  scope :funded, -> do
+    joins(:grant_submissions)
+      .where("grant_submissions.funding_decision == 't'")
```
Comment (ywwg) [inline app/models/artist.rb]: Is there any way to make this something like .where(grant_submissions.funding_decision: true) ?

Comment (Katee) [reply app/models/artist.rb]: Yep, you can use `.where("grant_submissions.funding_decision": true)`. I just copied the query as is.

Comment (Katee) [reply app/models/artist.rb]: You can also combine scopes from GrantSubmission.

Comment (Katee) [reply app/models/artist.rb]: Updated to use the existing `GrantSubmission.funded` scope.

**Comment by ywwg** [comment]
Nice.  the =='t' thing was the only way I could get it to work, and that's a Ruby implementation detail I don't want to put in code if I can avoid it.

**Comment by ywwg** [comment]
LGTM

**Pull request merged**